velox/docs/monthly-updates/july-2022.rst

```diff
@@ -0,0 +1,58 @@
+****************
+July 2022 Update
+****************
+
+Documentation
+=============
+
+* Add documentation for Presto related :doc:`/functions/json`.
+
+Core Library
+============
+
+* Add `ApproxMostFrequentStreamSummary` that approximately computes the top frequent values from a large stream.
```

Comment:
This is an implementation detail of the approx_most_frequent aggregate function and is not part of the public API. Perhaps, remove this.